applications/openshift/api-server/api_server_api_priority_gate_enabled/rule.yml

```diff
@@ -0,0 +1,44 @@
+documentation_complete: true
+
+prodtype: ocp4
+
+title: 'Enable the APIPriorityAndFairness feature gate'
+
+description: |-
+    To limit the rate at which the API Server accepts requests, make
+    sure that the API Priority and Fairness feature is enabled.
+
+rationale: |-
+    Using <tt>APIPriorityAndFairness</tt> feature provides a fine-grained way
+    to control the behaviour of the Kubernetes API server in an overload
+    situation.
```

Comment:
That's nice, but what is the security rationale?

Reply:
Fixed locally, will mark as resolved when I push new patches.